_content/papers/beyond-reasonable-doubt_vincent-eltschinger.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "Beyond Reasonable Doubt? A Note on Dharmakīrti
+and Scepticism"
+authors:
+  - "Vincent Eltschinger"
+external_url: "https://philpapers.org/archive/ELTBRD.pdf"
+drive_links:
+  - "https://drive.google.com/file/d/1GrO0-Iiz75FIchradGLURiEpuCr7Ze2I/view?usp=sharing"
+course: epistemology
+tags:
+  - scepticism
+  - stoicism
+  - logic
+year: 2020
+booktitle: "Buddhism and Scepticism: Historical, Philosophical, and Comparative Perspectives"
+pages: "37--53" 
```

Comment:
Also let's include `chapter: 3` as well (even though I know the chapters aren't "officially" numbered, but w/e...)